predator/asp/greedy-target-seed-solving_SA.lp

```diff
@@ -23,13 +23,6 @@ any_seed(S) :- seed(S).
 metabolite(M) :- reactant(M,_).
 metabolite(M) :- product(M,_).
 
-% try to cut the set of possible seeds
-can_reach(M) :- target(M).
-can_reach(M) :- reactant(M,R), product(P,R), can_reach(P).
-
-% % choose a set of seed from the possible seeds
-% { seed(M): can_reach(M), not forbidden(M) }.
-% located in the constraint/minimisation file
```

Comment:
As this is located in constraint/minimisation I remove this. can_reach is used nowhere else.